charts/redpanda/helpers.go

```diff
@@ -464,6 +464,27 @@ func StrategicMergePatch(overrides PodTemplate, original corev1.PodTemplateSpec)
 		overrideContainers[string(container.Name)] = container
 	}
 
+	if overrides.Spec.Volumes != nil && len(overrides.Spec.Volumes) > 0 {
```

Comment:
At this point, it's probably worth it to move this into a sharable form. 

You might be able to get away with implementing it in the bootstrap package and then re-exporting it from `helmette`?